samples/cpp/benchmark_app/README.md

```diff
@@ -99,11 +102,22 @@ Options:
     -stream_output              Optional. Print progress as a plain text. When specified, an interactive progress bar is replaced with a multiline output.
     -t                          Optional. Time, in seconds, to execute topology.
     -progress                   Optional. Show progress bar (can affect performance measurement). Default values is "false".
-    -shape                      Optional. Set shape for input. For example, "input1[1,3,224,224],input2[1,4]" or "[1,3,224,224]" in case of one input size.
+    -shape                      Optional. Set shape for network input. For example, "input1[1,3,224,224],input2[1,4]" or "[1,3,224,224]" in case of one input size.
+                                This parameter affect model input shape and can be dynamic. For dynamic dimensions use symbol `?` or '-1'. Ex. [?,3,?,?].
+                                For bounded dimensions specify range 'min..max'. Ex. [1..10,3,?,?].
+    -data_shape                 Required for networks with dynamic shapes. Set shape for input blobs.
+                                For example, "[1,3,224,224]" or "input1[1,3,224,224],input2[1,4]" in case of
+                                one input size. In case of several input sizes provide the same number for
+                                each input (except cases with 1 shape for 1 input): "[1,3,128,128][3,3,128,128][1,3,320,320]",
```

Comment:
minor: "except cases with 1 shape for 1 input" -> "except cases with **single** shape for **any** input"

Reply:
Updated